signal2html/core.py

```diff
@@ -58,7 +59,7 @@ def get_color(db, recipient_id):
     return color
 
 
-def get_sms_records(db, thread, addressbook, version=None):
+def get_sms_records(db, thread, addressbook, versioninfo=None):
```

Comment:
I think this argument isn't used anymore right?

Reply:
True, removing

Reply:
:+1: